src/main/java/mandel/earthquake/EarthquakeFrame.java

```diff
@@ -0,0 +1,117 @@
+        group.add(hourButton);
+        group.add(monthButton);
+
+        JPanel radioButtonPanel = new JPanel();
+        radioButtonPanel.add(hourButton);
+        radioButtonPanel.add(monthButton);
+
+        add(radioButtonPanel, BorderLayout.NORTH);
+
+        EarthquakeService service = new EarthquakeServiceFactory().getService();
+
+        hourButton.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                if (hourButton.isSelected()) {
+                    Disposable disposable = service.oneHour()
+                            // tells Rx to request the data on a background Thread
+                            .subscribeOn(Schedulers.io())
+                            // tells Rx to handle the response on Swing's main Thread
+                            .observeOn(SwingSchedulers.edt())
+                            //.observeOn(AndroidSchedulers.mainThread()) // Instead use this on Android only
+                            .subscribe(
+                                    (response) -> { currentResponse = response;
+                                        handleResponse(response); },
+                                    Throwable::printStackTrace);
+                }
+            }
+        });
+
+        monthButton.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                if (monthButton.isSelected()) {
+                    Disposable disposable2 = service.oneMonth()
+                            // tells Rx to request the data on a background Thread
+                            .subscribeOn(Schedulers.io())
+                            // tells Rx to handle the response on Swing's main Thread
+                            .observeOn(SwingSchedulers.edt())
+                            //.observeOn(AndroidSchedulers.mainThread()) // Instead use this on Android only
+                            .subscribe(
+                                    (response) -> { currentResponse = response;
+                                        handleResponse(response); },
+                                    Throwable::printStackTrace);
+                }
+            }
+        });
+
+        ListSelectionModel listSelectionModel = earthquakeList.getSelectionModel();
+        listSelectionModel.addListSelectionListener(new ListSelectionListener() {
+            @Override
+            public void valueChanged(ListSelectionEvent e) {
+                Feature feature = currentResponse.features[earthquakeList.getSelectedIndex()];
+                double lng = feature.geometry.coordinates[0];
+                double lat = feature.geometry.coordinates[1];
+                Runtime rt = Runtime.getRuntime();
+                String url = "http://maps.google.com/maps?q= " + lat + ", " + lng;
+                try {
+                    rt.exec("rundll32 url.dll,FileProtocolHandler " + url);
```

Comment:
This isn't how I advised that you do this.
This will only work on Windows, not Mac